share/models/registration.py

```diff
@@ -0,0 +1,41 @@
+from model_utils import Choices
+
+from django.conf import settings
+from django.db import models
+from django.utils.translation import ugettext as _
+
+
+class RegistrationForm(models.Model):
```

Comment:
This is kind of a dumb thing, `RegistationForm` is a bit misleading as `Form`s are a thing in Django apps. Maybe just call it `ProviderRegistration` or `RegistrationRequest`? I'm open to anything that doesn't have `Form` in the name 🎱